weblate/memory/machine.py

```diff
@@ -33,6 +33,10 @@ class WeblateMemory(MachineTranslation):
     accounting_key = "internal"
     do_cleanup = False
 
+    def cleanup_text(self, text, unit):
+        """Don't change anything."""
+        return text, {}
```

Comment:
This is same as `do_cleanup = False` flag does, should not be needed. (Same as in weblate/machinery/weblatetm.py).